src/explore-education-statistics-frontend/src/modules/table-tool/components/__tests__/__data__/tableData.ts

```diff
@@ -0,0 +1,313 @@
+      value: 'bfd88241-1130-4df8-9e49-a411618d082f',
+      label: '02/04/2021',
+      group: 'Default',
+      isTotal: false,
+      category: 'Date',
+    }),
+    new CategoryFilter({
+      value: '3f223187-a2aa-420c-a3d8-e2a94f77e4b5',
+      label: '02/06/2020',
+      group: 'Default',
+      isTotal: false,
+      category: 'Date',
+    }),
+  ],
+  rows: [
+    new TimePeriodFilter({
+      label: '2020 Week 23',
+      year: 2020,
+      code: 'W23',
+      order: 0,
+    }),
+    new TimePeriodFilter({
+      label: '2020 Week 24',
+      year: 2020,
+      code: 'W24',
+      order: 1,
+    }),
+    new TimePeriodFilter({
+      label: '2020 Week 25',
+      year: 2020,
+      code: 'W25',
+      order: 2,
+    }),
+    new TimePeriodFilter({
+      label: '2020 Week 26',
+      year: 2020,
+      code: 'W26',
+      order: 3,
+    }),
+  ],
+};
+
+export const testSelectedPublicationWithLatestRelease = {
+  id: '536154f5-7f82-4dc7-060a-08d9097c1945',
+  title: 'Test publication',
+  slug: 'test-publication',
+  selectedRelease: {
+    id: 'latest-release-id',
+    latestData: true,
+    slug: 'latest-release-slug',
+    title: 'Latest Release Title',
+  },
+  latestRelease: {
+    title: 'Latest Release Title',
+  },
+};
+
+export const testSelectedPublicationWithNonLatestRelease = {
+  id: '536154f5-7f82-4dc7-060a-08d9097c1945',
+  title: 'Test publication',
+  slug: 'test-publication',
+  selectedRelease: {
+    id: 'selected-release-id',
+    latestData: false,
+    slug: 'selected-release-slug',
+    title: 'Selected Release Title',
+  },
+  latestRelease: {
+    title: 'Latest Release Title',
+  },
+};
+
+export const testPublication = {
+  id: '',
+  title: '',
+  yearTitle: '',
+  coverageTitle: '',
+  releaseName: '',
+  published: '',
+  slug: '',
+  summarySection: {
+    id: '',
+    order: 0,
+    heading: '',
+    content: [],
+  },
+  keyStatisticsSection: {
+    id: '',
+    order: 0,
+    heading: '',
+    content: [],
+  },
+  keyStatisticsSecondarySection: {
+    id: '',
+    order: 0,
+    heading: '',
+    content: [],
+  },
+  headlinesSection: {
+    id: '',
+    order: 0,
+    heading: '',
+    content: [],
+  },
+  publication: {
+    id: '',
+    slug: '',
+    title: '',
+    description: '',
+    dataSource: '',
+    summary: '',
+    otherReleases: [],
+    legacyReleases: [],
+    topic: {
+      theme: {
+        title: '',
+      },
+    },
+    contact: {
+      teamName: 'The team name',
+      teamEmail: 'team@name.com',
+      contactName: 'A person',
+      contactTelNo: '012345',
+    },
+    methodology: {
+      id: 'm1',
+      slug: 'm1',
+      summary: 'words',
+      title: 'methodology title',
+    },
+  },
+  latestRelease: true,
+  relatedInformation: [],
+  type: {
+    id: '',
+    title: ReleaseType.NationalStatistics,
+  },
+  updates: [],
+  content: [],
+  downloadFiles: [],
+  dataLastPublished: '',
+  hasPreReleaseAccessList: true,
+  hasMetaGuidance: true,
+};
```

Comment:
Where's all this stuff from? Don't think we need any of this?

Reply:
Types added as discussed in slack.